eogrow/pipelines/export_maps.py

```diff
@@ -235,51 +212,142 @@ def _prepare_files(self, geotiff_paths: List[str], output_file: str) -> Tuple[FS
             temp_geotiff_paths,
             workers=None,
             multiprocess=False,
-            desc="Making local copies",
         )
-        temp_map_path = fs.path.basename(output_file)
-        return temp_fs, temp_geotiff_paths, temp_map_path
+        return temp_fs, temp_geotiff_paths
+
+    def _split_patches_temporally(
+        self, filesystem: FS, output_folder: str, geotiff_paths: List[str], some_eopatch: str, crs: CRS
+    ) -> Dict[Optional[dt.datetime], List[str]]:
+        """Splits input patches into multiple tiffs according to the temporal dimension.
+
+        Here the jobs are grouped together per-inputfile, in later parts jobs are grouped per-time.
+        Due to this jobs are created per-input, but we also take note of per-time grouping, which the function returns.
+        """
+        LOGGER.info("Splitting TIFF files temporally.")
+        num_bands, timestamp = self._load_bands_and_timestamp(some_eopatch)  # assume eopatches share these
+
+        def get_extraction_path(input_path: str, time: dt.datetime) -> str:
+            """Ensures that after extraction the files have a time-suffix."""
+            tiff_name = _strip_tiff_extension(fs.path.basename(input_path))
+            return fs.path.join(output_folder, get_tiff_name(self.map_name, tiff_name, crs, time))
+
+        geotiffs_per_time: Dict[Optional[dt.datetime], List[str]] = defaultdict(list)
```

Comment:
because in L245 i'm appending directly to an element of the `geotiffs_per_time`. Were it a normal dict i'd have to dance around with checks if something is inside the dict and if not to insert an empty list etc.